appservice/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@microsoft/vscode-azext-azureappservice",
     "author": "Microsoft Corporation",
-    "version": "0.1.10",
+    "version": "0.1.11",
```

Comment:
Since it's a dev dependency, we don't need to release a new version to clear out the CG alert.